CHANGES.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased
 
+* Fix: Cobertura.branch_rate() returns None when branch rate is absent instead of raising TypeError
```

Comment:
```suggestion
* Fix: `Cobertura.branch_rate()` returns `None` when `branch-rate` is absent from the Cobertura coverage report instead of raising `TypeError`. Thanks @starcruiseromega
```